app/lib/brexit_checker/account_jwt.rb

```diff
@@ -0,0 +1,33 @@
+class BrexitChecker::AccountJwt
+  def initialize(criteria_keys)
+    @criteria_keys = criteria_keys
+  end
+
+  def encode(key = ecdsa_key, algorithmn = "ES256")
+    JWT.encode payload, key, algorithmn
+  end
+
+private
+
+  attr_reader :criteria_keys
+
+  def payload
+    {
+      uid: uid,
+      data: criteria_keys,
```

Comment:
This should be `attributes: { transition_checker_state: criteria_keys }`

There also needs to be a `key` field (with a UUID value) which identifies the signing key used

Reply:
Something for the future: we could provide an accounts test helper which includes things like a JWT validator

Reply:
amended [here](https://github.com/alphagov/finder-frontend/pull/2207/commits/e43d83b52b5813a84303a4ecc93f047fc719a2ad#diff-e4257d6c3bb3ff33ad32620a2d04b871R14-R21)